strings/src/main/res/values/strings.xml

```diff
@@ -196,7 +196,12 @@
 
     <!-- Error message displayed when copying media from a temporary directory fails -->
     <string name="answer_file_copy_failed_message">Media could not be attached to form but is available at %1$s</string>
-    
+
+    <string name="saving_audio_recording_failed">Saving audio recording failed</string>
```

Comment:
I don't think this belongs here.

Reply:
It felt related to the answer file copy failure to me. Where do you think it should go?

Reply:
Isn't it a new string that's not actually used?!

Reply:
huh I see! Hmmm I'll fix, but I'm weirded out that lint didn't catch this...